src/assets/BrowserAutomationProtocol/browser-automation-protocol.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Different Approaches (Protocols) to Automate the Browser
+authorName: Sawjan Gurung
+authorAvatar: https://www.jankaritech.com/images/2021/12/29/p1110923-crop-col-800.jpg
+authorLink: https://github.com/saw-jan
+createdAt: 22/11/2022
+tags: browser automation, testing, playwright, selenium, webdriver, cypress, protocols, chrome devtools
+banner: /src/assets/BrowserAutomationProtocol/images/cover.jpg
```

Comment:
@kiranparajuli589  will this local link work as banner url?

Reply:
I think it doesn't :smile: 
https://github.com/JankariTech/blog/actions/runs/3529586629/jobs/5920730239

Reply:
It would be nice if we can use the local banner image. Need a new feature?
CC @kiranparajuli589 @grgprarup

Reply:
Ahh, noice :+1:

> surely this feature needs optimizations for ease of access

Yeah, I would prefer to use `/src/assets/...` paths in the blogs for local assets and after build the url path would resolve to `https://<domain>/src/assets/...`. Because the domain could be different.

Reply:
:100:  true... I will quickly make a ticket for this.

Reply:
https://github.com/JankariTech/blog/issues/67

Reply:
will think about it :smile: